apps/smartling/lambda/serverless.yml

```diff
@@ -26,12 +25,11 @@ provider:
   deploymentPrefix: sls-apps-smartling
 
 package:
-  include:
-    - built/**
-  exclude:
-    - .git/**
-    - "**/*.ts"
-    - node_modules/smartling-frontend/node_modules/**
+  patterns:
+    - 'built/**'
```

Comment:
Is it just me or is this much less readable than how it worked before

Reply:
🤷 I think it's alright. It now feels more like a glob pattern and not a custom implementation.